packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -682,21 +724,65 @@ test('price falls precipitously', async t => {
   await manualTimer.tick();
   await waitForPromisesToSettle();
   // An emergency liquidation got less than full value
-  const newDebtAmount = await E(vault).getCurrentDebt();
+  const debtAfterLiquidation = await E(vault).getCurrentDebt();
   t.truthy(
-    AmountMath.isGTE(AmountMath.make(runBrand, 70n), newDebtAmount),
-    `Expected ${newDebtAmount.value} to be less than 70`,
+    AmountMath.isGTE(AmountMath.make(runBrand, 70n), debtAfterLiquidation),
+    `Expected ${debtAfterLiquidation.value} to be less than 70`,
   );
 
   t.deepEqual(await E(vaultFactory).getRewardAllocation(), {
     RUN: AmountMath.make(runBrand, 19n),
   });
 
-  const liquidations = await E(
-    E(vault).getLiquidationSeat(),
-  ).getCurrentAllocation();
-  t.deepEqual(liquidations.Collateral, AmountMath.make(aethBrand, 1n));
-  t.deepEqual(liquidations.RUN, AmountMath.makeEmpty(runBrand));
+  t.deepEqual(
+    await E(vault).getCollateralAmount(),
+    // ??? why is there any collateral left when there's still debt?
```

Comment:
@Chris-Hibbert can you help me to understand this? I thought liquidation used up collateral until debt was covered (or there was none left)

Reply:
Good catch!

liquidation wants to use up the collateral until the debt is covered. The AMM is giving the liquidator a price break here.

The liquidator wasn't able to get enough for the 400 aEth to cover the 398 RUN debt, so it said to the AMM "sell the 400 aEth for whatever we can get".  The AMM said "you can get 325 RUN for that, but you only have to pay 399 aEth to get 325 RUN", so it refunded one aEth. 

Since there's a remaining debt, the vaults should keep the extra collateral rather than returning it. We now have enough mechanism in place that it makes sense to hold onto it. In fact, when @dtribble's batched liquidation comes in, we'll be able to add the unsold collateral to what remains, and offer it with the next tranche.